Jenkinsfile

```diff
@@ -146,4 +146,9 @@ pipeline {
       }
     }
   }
+  post {
```

Comment:
Just checking will this post script run even if the build fails? we don't want to loose any build logs if the build fails.